### ISSUE TITLE
Can't build WinUI class library in Release in Reunion 0.8

### ISSUE THREAD

**Issue by huoyaoyuan** [microsoft/microsoft-ui-xaml#5113]
**Describe the bug**
PriGen expects wrong file position and fails.

**Steps to reproduce the bug**

1. Create a WinUI class library project.
2. **This version doesn't support AnyCPU, however the template project is still AnyCPU**. Manually add platforms and fix sln to continue.
3. Add a user control. Verify it can build in Debug.
4. Switch to release. The build will fail with:
```
...Microsoft.ApplicationModel.Resources.PriGen.targets(760,5): error MSB3030: Can't copy file "...\WinUILib\bin\x86\Release\net5.0-windows10.0.19041.0\UserControl1.xbf" because it doesn't exist.
```

The actual output layout is:
```
bin
- x86
  - Debug
    - net5.0-windows10.0.19041.0
      - WinUILib
        - UserControl1.xbf
  - Release
    - net5.0-windows10.0.19041.0
      - WinUILib
        - UserControl1.xbf
```

The output layouts are identical for Debug and Release, but wrong layout passed to PriGen in Release.

**Expected behavior**
Build behaviors the same.

**Version Info**

NuGet package version: 
[WinUI 3 - Project Reunion 0.8 Preview: 0.8.0-preview] 

Windows app type:
| UWP              | Win32            |
| :--------------- | :--------------- |
|  | Yes |

**Comment by huoyaoyuan** [comment]
The output layout was different in Debug and Release in previous. This may be a side effect when unifying them.

**Comment by anawishnoff** [comment]
Thanks for filing @huoyaoyuan - the team is looking into this issue.

**Comment by huoyaoyuan** [comment]
Looks resolved in 0.8 stable.
